addons/viewport/src/shortcuts.ts

```diff
@@ -0,0 +1,60 @@
+import { API } from '@storybook/api';
+import { ADDON_ID } from './constants';
+
+const getCurrentViewportIndex = (viewportsKeys: string[], current: string): number =>
+  viewportsKeys.indexOf(current);
+
+const getNextViewport = (viewportsKeys: string[], current: string): string => {
+  const currentViewportIndex = getCurrentViewportIndex(viewportsKeys, current);
+  return currentViewportIndex === viewportsKeys.length - 1
+    ? viewportsKeys[0]
+    : viewportsKeys[currentViewportIndex + 1];
+};
+
+const getPreviousViewport = (viewportsKeys: string[], current: string): string => {
+  const currentViewportIndex = getCurrentViewportIndex(viewportsKeys, current);
+  return currentViewportIndex < 1
+    ? viewportsKeys[viewportsKeys.length - 1]
+    : viewportsKeys[currentViewportIndex - 1];
+};
+
+export const registerShortcuts = async (api: API, setState: any, viewportsKeys: string[]) => {
+  await api.setAddonShortcut(ADDON_ID, {
+    label: 'Previous viewport',
+    defaultShortcut: ['shift', 'V'],
+    actionName: 'previous',
+    action: () => {
+      const { selected, isRotated } = api.getAddonState(ADDON_ID);
+      setState({
+        selected: getPreviousViewport(viewportsKeys, selected),
+        isRotated,
+      });
+    },
+  });
+
+  await api.setAddonShortcut(ADDON_ID, {
+    label: 'Next viewport',
+    defaultShortcut: ['V'],
+    actionName: 'next',
+    action: () => {
+      const { selected, isRotated } = api.getAddonState(ADDON_ID);
+      setState({
+        selected: getNextViewport(viewportsKeys, selected),
+        isRotated,
+      });
+    },
+  });
+
+  await api.setAddonShortcut(ADDON_ID, {
+    label: 'Reset viewport',
+    defaultShortcut: ['alt', 'V'],
```

Comment:
This doesn't work for me on macOS/Chrome.

When I override the shortcut, instead of `V`, it shows `√`.

![Screen Shot 2021-04-29 at 1 46 57 PM](https://user-images.githubusercontent.com/85783/116616323-71841b80-a8f1-11eb-971c-dff419d4a387.png)

Video:

https://user-images.githubusercontent.com/85783/116616253-55807a00-a8f1-11eb-8786-eaf409d05808.mov

Reply:
I'm very sorry for that, Kaelig 😞 Is this <kbd>alt</kbd> + <kbd>v</kbd> combination a macOS's built-in shortcut?

Reply:
Yes, the option key is very commonly used in macOS to provide alternative characters or alternative key functionality (which is a feature I love and I miss a lot when I use Windows).